## Service 
geometry_msgs/Pose2D[] poses
---
bool path_safe